packages/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.kt

```diff
@@ -325,7 +323,15 @@ class RNGestureHandlerButtonViewManager : ViewGroupManager<ButtonViewGroup>(), R
         colorDrawable.setCornerRadii(buildBorderRadii())
       }
 
-      val layerDrawable = LayerDrawable(if (selectable != null) arrayOf(colorDrawable, selectable, borderDrawable) else arrayOf(colorDrawable, borderDrawable))
+      val layerDrawable = LayerDrawable(
+        if (selectable !=
+          null
+        ) {
```

Comment:
🤦, can we have it on a single line, or is the new spotless that special?

Reply:
Well, it is special in some cases 😅 I've went through changed files and fixed some of those issues, let me know if there's anything else that should be changed. ([9cf0cb9](https://github.com/software-mansion/react-native-gesture-handler/pull/3507/commits/9cf0cb9e1f3c7af79c8fa717dc98530ae1d087cb))